README.md

```diff
@@ -64,7 +64,7 @@ This is the simplest case. Get the script in whatever way you prefer and include
 
 ```html
 <script src="//path/to/video.min.js"></script>
-<script src="//path/to/videojs-concat.min.js"></script>
+<script src="//path/to/plugin-concat.min.js"></script>
```

Comment:
I think this should be `videojs-plugin-concat` which can be done by changing the `distName` option in `rollup.config.js` to `videojs-plugin-concat`